### ISSUE TITLE
what should i change if i want to use user defined ROI (like the rect in LucasKanade) in the affine lucas-kanade

### ISSUE THREAD

**Issue by wanghetongtt** [ziliHarvey/Lucas-Kanade-tracking-and-Correlation-Filters#1]

**Comment by ziliHarvey** [comment]
Basically, you can modify [rect](https://github.com/ziliHarvey/Lucas-Kanade-tracking-and-Correlation-Filters/blob/master/src/testCarSequence.py#L9), and also please modify the [tracker parameters](https://github.com/ziliHarvey/Lucas-Kanade-tracking-and-Correlation-Filters/blob/master/src/LucasKanade.py#L31). These tracker parameters are essentially width and height, sorry for not making it modular.

**Comment by ziliHarvey** [comment]
**LucasKanadeAffine is used in dominant motion estimation to derive the transformation matrix M that warps a previous image to the current frame**, which takes the entire image L_t as the template to be tracked in image L_(t+1), this approach is reasonable under the assumption that a majority of the pixels correspond to the stationary objects in the scene whose depth variation is small relative to their distance from camera.

**In short, for LucasKanadeAffine, there's no rect since the entire Lt image is the template.**, you are welcomed to implement it as a LucasKanade version which takes into a L_t, L_(t+1), and a rect template.
